Symfony/CS/Tests/Fixer/Contrib/AlignDoubleArrowFixerTest.php

```diff
@@ -30,6 +30,77 @@ public function provideFixCases()
     {
         return array(
             array(
+                '$array = array(
```

Comment:
- missing PHP opening tag
- missing enter after comma to be consistent with other cases
- missing indent for array content

```php
            array(
                '<?php
    $array = array(
        "closure" => function ($param1, $param2) {
            return;
        }
    );',
            ),
```